src/js/components/BrimToaster.tsx

```diff
@@ -1,37 +1,25 @@
-import {Toaster, useToasterStore} from "react-hot-toast"
-import React, {useEffect, useState} from "react"
-import last from "lodash/last"
-import isEmpty from "lodash/isEmpty"
-import styled from "styled-components"
-import {toastLocator} from "../../../test/playwright/helpers/locators"
+import env from "app/core/env"
+import React from "react"
+import {Toaster} from "react-hot-toast"
 
-const HiddenToast = styled.p`
-  display: none;
-`
-
-/*
-  BrimToaster renders the react-hot-toaster component and also caches the most recent
-  "toast" generated and renders it in a persisted, hidden dom element so that we can more
-  reliably test for it's existence in our integration tests
- */
 const BrimToaster = () => {
-  const [lastToast, setLastToast] = useState(null)
-  const {toasts} = useToasterStore()
-
-  useEffect(() => {
-    if (isEmpty(toasts)) return
-
-    setLastToast(last(toasts))
-  }, [toasts])
-
   return (
-    <>
-      <HiddenToast {...toastLocator.props}>{lastToast?.message}</HiddenToast>
-      <Toaster
-        position="bottom-right"
-        toastOptions={{className: "brim-toast"}}
-      />
-    </>
+    <Toaster
+      position="bottom-right"
+      toastOptions={{
+        className: "brim-toast",
+        duration: env.isTest ? 2 ** 31 - 1 : undefined,
+        success: {
+          duration: env.isTest ? 2 ** 31 - 1 : undefined
+        },
+        loading: {
+          duration: env.isTest ? 2 ** 31 - 1 : undefined
+        },
+        error: {
+          duration: env.isTest ? 2 ** 31 - 1 : undefined
+        }
+      }}
```

Comment:
Don't delete the toaster toasts if we're testing. This could actually be mitigated with fake timers.